### DISCUSSION TITLE
Can we trust that browser tool? :-D

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#92] [General]
LightHouse Test is showing the following on Mobile phone : 

I can see that images for icons are in png format which is always slower format to load even if the size of the image is low. may be webp or jpg ??
![Screenshot 2023-05-26 at 20 11 20](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/a470d25d-8416-4109-a797-a8b51840f746)
![Screenshot 2023-05-26 at 20 12 14](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/2d73f140-d0b1-4fce-9faa-3dc7af6b5660)
![Screenshot 2023-05-26 at 20 13 00](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/713e0c71-d4f2-4b46-809c-3181e90d7771)

also some unused javascript could be avoided ?

![Screenshot 2023-05-26 at 20 12 28](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/f98ae5c8-f856-4c3f-9134-14c34f017734)

**Comment by TJ-BitCanna** [comment]
We are aware of a revisit on the icons indeed :)

The optimalisations we'll take a good look at.

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- recheck unused Javascript code
- define what is needed to speed up loading times of the WeedWallet

**Comment by TJ-BitCanna** [comment]
https://github.com/BitCannaGlobal/WeedWallet/issues/113#issuecomment-1580656706
